cpp/ironoxide.pc.in

```diff
@@ -0,0 +1,10 @@
+prefix=%PREFIX%
```

Comment:
Is there a reason this is `.pc.in` instead of just `.pc`?

Reply:
This is a file that is not a valid .pc file. It's an input to a replace which replaces the %PREFIX% with the actual prefix requested.

Reply:
Ah, ya. Well blerg. In order to get Swift SDK working on Linux/Mac we need a valid one of these files that we can symlink in. What does this `PREFIX` get replaced with and where does that happen? I see the `sed` command above, but it wasn't obvious to me that it was replacing this.

Reply:
It's actually in the homebrew recipe. When homebrew installs things it seems that you can choose where the things go or something. I'm not aware of all the details.

The line that does the replace is here: https://github.com/IronCoreLabs/homebrew-ironcore/pull/1/files#diff-fd867bcea19121b4c3b90032ac8336c9R13